### ISSUE TITLE
🐞 Bug report: Image Box title colour being overridden using inherit

### ISSUE THREAD

**Issue by leecollings-sb** [elementor/elementor#18100]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When the option to disable Elementor styling is enabled, there is an inline styling rule that set the colour of the Image Box title link to inherit, which is overriding the stylesheet being used in my theme.

I would expect this to be removed when using the option to disable styling within Elementor (so that I can use my theme's stylesheet).

### Steps to reproduce

Check 'Disable Default Colours' in Elementor settings.
Add an Image Box on any page (with link).
When viewing the styles for the title element, the following rule is being applied:
```
.elementor-widget-image-box .elementor-image-box-title a {
    color: inherit;
}
```

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

					
== Server Environment ==
	Operating System: Darwin
	Software: Apache/2.4.46 (Unix) mod_fastcgi/mod_fastcgi-SNAP-0910052141 PHP/7.4.27 OpenSSL/1.0.2u mod_wsgi/3.5 Python/2.7.18
	MySQL version: MySQL Community Server (GPL) v5.7.34
	PHP Version: 7.4.27
	PHP Max Input Vars: 1000
	PHP Max Post Size: 1024M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.9.1
	Site URL: http://new.nirvanalight.local
	Home URL: http://new.nirvanalight.local
	WP Multisite: No
	Max Upload Size: 1 GB
	Memory limit: 256M
	Permalink Structure: /blog/%postname%/
	Language: en-GB
	Timezone: Europe/London
	Admin Email: webadmin@supportedbusiness.com
	Debug Mode: Inactive

== Theme ==
	Name: Nirvana Light
	Version: 
	Author: Lee Collings
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.109 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.5.6
		Author: Elementor.com

	InverOak Horoscopes
		Version: 1.2
		Author: InverOak

	InverOak Members Helper
		Version: 1.1
		Author: InverOak

	InverOak Operator Feed
		Version: 1.25.8
		Author: InverOak

	InverOak WebCam/Chat
		Version: 1.5
		Author: InverOak

	New RoyalSlider
		Version: 3.4.2
		Author: Dmitry Semenov

	WooCommerce
		Version: 6.2.1
		Author: Automattic

	Yoast SEO
		Version: 18.3
		Author: Team Yoast

== Elementor Experiments ==
	Optimised DOM Output: Active by default
	Improved asset loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility improvements: Active by default
	Import Export Template Kit: Active by default
	Additional Custom Breakpoints: Active by default
	Hide native WordPress widgets from search results: Active by default
	Landing pages: Active by default
	Color Sampler: Active by default
	Favorite Widgets: Active by default
	Admin Top Bar: Active by default

== Log ==
	

== Elementor - Compatibility Tag ==

**Comment by rami-elementor** [comment]
Hi @XXX,

Thank you for submitting a feature request.

After review, we have decided not to change this behavior as it is not a bug. If the color is not set, it should be inherited.

You can have a higher specificity CSS rule in your theme to override the default color:

```
.elementor-widget-image-box .elementor-image-box-title a {
    color: red;
}
```

Regards,

**Comment by leecollings-sb** [comment]
@rami-elementor I disagree, it is a bug because it now means the colour is NOT inheriting the colour I've set to a elements in my themes stylesheet.

The point of disabling the styling, is surely to allow me to use my theme stylesheet?? Which this is stopping me from doing.

Please re-open and fix.

**Comment by rami-elementor** [comment]
@leecollings-sb

**Comment by leecollings-sb** [comment]
@rami-elementor This screenshot shows a page that's using Image Box. The inspector is showing the `color:inherit;` declaration which is overriding my theme styling.

The only way around this is to add `!important` to the CSS declaration. However, if I was using a theme that didn't allow custom CSS, this would never be overridden. That is why this inherit line should be removed, it is not being helpful in this situation (and negates the whole point of trying to disable Elementor's default styling).

![Screenshot 2023-12-14 at 16 10 20](https://github.com/elementor/elementor/assets/12139008/5640fdae-ea3b-4d01-a1a4-72a6a6dde6cd)
